tests/common/plugins/sanity_check/__init__.py

```diff
@@ -119,8 +130,51 @@ def do_checks(request, check_items, *args, **kwargs):
     return check_results
 
 
+@pytest.fixture(scope="module", autouse=True)
+def log_custom_msg(request):
+    yield
+    module_name = request.node.name
```

Comment:
Can we confirm the producer of the keys running before log_custom_msg?
In another word, should we make sure the destruction of log_custom_msg is always the last running one?

Reply:
![image](https://github.com/user-attachments/assets/8127ceb6-a810-4ece-b18d-480e3110896e)

Reply:
log_custom_msg will yield after all the checks, including sanity_check and core_dump_and_config_check

Reply:
Also, its now included in the sanity_check fixture, teardown will always happen after all the sanity_check.